### DISCUSSION TITLE
Azuracast URL change

### DISCUSSION THREAD

**Discussion by BalderdashBoys** [AzuraCast/AzuraCast#6018] [Q&A]
I have a domain that I would like to use that would point to my live radio public page. I am not a coder and need a dumbed down step-by-step help.
I have no idea where to put it or what nameservers I need the domain to point to. 
I use Linode as my server, which azuracast is installed.
All answers I have found so far are tailored for coding tech people, which I am not. 
Thank you in advance!

**Comment by Mozitux** [comment]
Hello,
In the management interface of your domain name provider, you must enter the IP address of your Azuracast server in the **"A" Record** part.

It should look like this:
![azu_like_this](https://user-images.githubusercontent.com/19860756/212882705-378ea197-4daf-4882-8a96-eee8bcf0da69.PNG)
